rs/nervous_system/integration_tests/tests/sns_release_qualification.rs

```diff
@@ -279,3 +280,95 @@ async fn test_sns_upgrade(sns_canisters_to_upgrade: Vec<SnsCanisterType>) {
         .await;
     }
 }
+
+#[tokio::test]
+async fn test_proper_sns_swap_upgrade() {
```

Comment:
Are you going to change the existing tests and remove this test eventually?  I thought we discussed that yesterday, since the other tests could theoretically already handle this with some slight modifications.

I'd make a TODO note if that's being blocked here.

Reply:
Added a DO NOT MERGE and an explanation to the comment above this test.